astropy/table/table.py

```diff
@@ -2021,6 +2021,70 @@ def group_by(self, keys):
 
         return groups.table_group_by(self, keys)
 
+    def to_pandas(self):
+        """
+        Return a :class:`pandas.DataFrame` instance
+
+        Returns
+        -------
+        dataframe : :class:`pandas.DataFrame`
+            A pandas :class:`pandas.DataFrame` instance
+        """
+        from pandas import DataFrame
+
+        if self.has_mixin_columns:
+            raise ValueError("Cannot convert a table with mixin columns to a pandas DataFrame")
+
+        for column in self.columns:
```

Comment:
@mhvk - the error that Pandas gives is pretty cryptic:
```
In [13]: t = Table([[[1,2]]])
In [14]: print(t)
col0 [2]
--------
  1 .. 2
In [15]: pandas.DataFrame(np.array(t))
...
ValueError: If using all scalar values, you must must pass an index
```
I think pre-catching the exception is better.  Also, I don't think support for multi-d columns is coming any time soon.  I think this would cut rather deeply into the fundamental data structures (though I'm not an expert in Pandas so I could be wrong).  I asked Wes about this several years ago and the answer was hmm... would have to think about that...